src/frontend/screens/Settings/CreateOrJoinProject/index.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import {MessageDescriptor, defineMessages, useIntl} from 'react-intl';
+import {Text} from '../../../sharedComponents/Text';
+import {NativeNavigationComponent, ViewStyleProp} from '../../../sharedTypes';
+import {ScrollView, StyleSheet, View} from 'react-native';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import {LIGHT_GREY} from '../../../lib/styles';
+import Warning from '../../../images/Warning.svg';
+import {useAllProjects} from '../../../hooks/server/projects';
+import {CenteredView} from '../../../sharedComponents/CenteredView';
+import {Loading} from '../../../sharedComponents/Loading';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Settings.CreateOrJoinProject.title',
+    defaultMessage: 'Create or Join',
+  },
+  createProject: {
+    id: 'screens.Settings.CreateOrJoinProject.createProject',
+    defaultMessage: 'Create a Project',
+  },
+  startProject: {
+    id: 'screens.Settings.CreateOrJoinProject.startProject',
+    defaultMessage: 'Start a CoMapeo Project',
+  },
+  joinProject: {
+    id: 'screens.Settings.CreateOrJoinProject.joinProject',
+    defaultMessage: 'Join a Project',
+  },
+  joinExisting: {
+    id: 'screens.Settings.CreateOrJoinProject.joinExisting',
+    defaultMessage: 'Join an existing Mapeo Project',
+  },
+  whatIsAProject: {
+    id: 'screens.Settings.CreateOrJoinProject.whatIsAProject',
+    defaultMessage: 'What is a Project',
+  },
+  projectDescription: {
+    id: 'screens.Settings.CreateOrJoinProject.projectDescription',
+    defaultMessage:
+      'A project is a secure container for your data. Only devices you invite can enter and share data with you. Create or Join a project in order to share data with other devices.',
+  },
+  alreadyOnProject: {
+    id: 'screens.Settings.CreateOrJoinProject.alreadyOnProject',
+    defaultMessage:
+      'You are already on a project. To create a new Project you must uninstall and reininstall CoMapeo.',
+  },
+});
+
+export const CreateOrJoinProject: NativeNavigationComponent<
+  'CreateOrJoinProject'
+> = ({navigation}) => {
+  const {formatMessage: t} = useIntl();
+  const projects = useAllProjects();
+
+  if (projects.data) {
+    return (
+      <ScrollView style={styles.container}>
+        <View style={styles.greyBox}>
+          <Text style={{fontWeight: 'bold'}}>{t(m.whatIsAProject)}</Text>
+          <Text>{t(m.projectDescription)}</Text>
+        </View>
+        {projects.data.length > 1 ? (
```

Comment:
what's the more proper check for this? checking if there exists a project is not a "default" project? If needed, maybe add a todo comment to note that this needs to be revisited

Reply:
Yeah, maybe ill open an issue. This was the workaround for now given that we only want the user to (seemingly) be only on one project. Since there is no default project key, this was the best way to determine that.